src/Microsoft.AspNet.OData/Formatter/ODataMediaTypeFormatter.cs

```diff
@@ -335,6 +344,16 @@ public override Task<object> ReadFromStreamAsync(Type type, Stream readStream, H
             }
         }
 
+        private static async Task CopyStreamAsync(Stream source, Stream destination)
+        {
+            if(source != null)
```

Comment:
`if (source != null)` - space is missing.

Reply:
Fixed, thanks @NicholasNoise